package.json

```diff
@@ -59,6 +58,33 @@
     "plugin:build:commit": "lerna run build:commit --ignore=\"@grafana/*\" --ignore=\"@grafana-plugins/input-datasource\"",
     "plugin:build:dev": "lerna run dev --ignore=\"@grafana/*\" --ignore=\"@grafana-plugins/input-datasource\""
   },
+  "nx": {
```

Comment:
I vaguely remember the NX config by now, but can we move this to the NX.json file instead?

Reply:
Because of the way top-level scripts work with NX it doesn't seem we can put this info in the nx.json file. I've moved it over to a project.json file and cleaned up the repetition.